include/ListView/ListViewColumnsFilterDialog.tpl

```diff
@@ -52,10 +52,10 @@
 
                 </div>
                 <div class="modal-footer">
-                    <button type="button" class="btn button purple btn-default"
-                            data-dismiss="modal">{$APP.LBL_CLOSE_BUTTON_TITLE}</button>
-                    <button type="button" onclick="columnsFilter.onSaveClick();" type="button"
-                            class="button red">{$APP.LBL_SAVE_CHANGES_BUTTON_TITLE}</button>
+                    <input title="{$APP.LBL_CLOSE_BUTTON_TITLE}" type="button" class="button"
```

Comment:
Hi @serhiisamko091184 why are we changing from button to input?

Reply:
Hello @clemente-raposo 
thanks for question and reviewing my PR, decision to use inputs was due to:
1) neibourhood Filter uses Inputs in Filter PopUp (so to make it a bit consistent, and many ather btns use <inputs> mostly - on create, edit, subpanels etc.); 
![image](https://github.com/salesagility/SuiteCRM/assets/116086008/85291345-ba47-4317-b861-ca24e82ab62b)

2) It was a bit messy, with exactly html element <button>, to apply CSS classes to get hovering and styling I wanted. 

It's possible to do with buttons, do I need to make changes? 

Thanks. 

Regards, 
Serhii